### ISSUE TITLE
White-list keyword arguments

### ISSUE THREAD

**Issue by georgwiese** [scalableminds/cluster_tools#15]
`get_executor(environment, *args, **kwargs)` fails if keyword args are passed that the executor doesn't understand. These arguments should be discarded so that the caller doesn't have to worry about passing the right arguments (it would just pass all arguments that could be relevant).

It might make sense to explicitly white-list arguments for each executor so that in the future, we can also do a translation between different executors (e.g. slurm and PBS (#14)).

This issue is blocking https://github.com/scalableminds/segmentation-tools/issues/421

**Comment by philippotto** [comment]
I released v1.3 which should unblock you. In the long run, we might want to think about an interface like this, though:

```
exc = get_executor(env, { "slurm": { ... }, "multiprocessing": { ... }, ... })
```

Right now, we are mixing all args together (brittle against name clashes) and separate them again.

**Comment by georgwiese** [comment]
+1 for that.
